.github/CODEOWNERS

```
@@ -497,6 +497,7 @@ utils/bazel/llvm-project-overlay/libc/** @llvm/pr-subscribers-libc
 /llvm/**/*RISCV* @llvm/pr-subscribers-riscv
 /clang/**/*RISCV* @llvm/pr-subscribers-riscv
 /clang/include/clang/Basic/riscv_*.td @llvm/pr-subscribers-riscv
+/clang/lib/Headers/riscv_* @llvm/pr-subscribers-riscv
```

Any reason not to do:

```suggestion
/llvm/**/*RISCV* @llvm/pr-subscribers-riscv
/llvm/**/*riscv* @llvm/pr-subscribers-riscv
/clang/**/*riscv* @llvm/pr-subscribers-riscv
/clang/**/*RISCV* @llvm/pr-subscribers-riscv
/lld/**/*riscv* @llvm/pr-subscribers-riscv
/lld/**/*RISCV* @llvm/pr-subscribers-riscv
```
?

---

I was wondering if lld RISC-V should be it's own team? Seems like a much smaller set of interested people.

---

I'd be ok with that

---

See also https://discourse.llvm.org/t/input-needed-teams-for-pull-request-subscriptions/73116/80

There is a difference between “core” backend files and catch-all patterns.

---

@MaskRay Do you have a suggestion for what "core" for RISC-V should be?